### ISSUE TITLE
`watch` panics with no such file

### ISSUE THREAD

**Issue by vn-ki** [rust-lang/rustlings#173]
I ran `rustlings watch` and it works. But when I start editing, it panics and exits.

Note: I am running it inside the repo.

```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or direct
ory" }', src/libcore/result.rs:997:5                                                                                                 
stack backtrace:
   0: std::sys::unix::backtrace::tracing::imp::unwind_backtrace
             at src/libstd/sys/unix/backtrace/tracing/gcc_s.rs:39
   1: std::sys_common::backtrace::_print
             at src/libstd/sys_common/backtrace.rs:71
   2: std::panicking::default_hook::{{closure}}
             at src/libstd/sys_common/backtrace.rs:59
             at src/libstd/panicking.rs:197
   3: std::panicking::default_hook
             at src/libstd/panicking.rs:211
   4: std::panicking::rust_panic_with_hook
             at src/libstd/panicking.rs:474
   5: std::panicking::continue_panic_fmt
             at src/libstd/panicking.rs:381
   6: rust_begin_unwind
             at src/libstd/panicking.rs:308
   7: core::panicking::panic_fmt
             at src/libcore/panicking.rs:85
   8: core::result::unwrap_failed
   9: rustlings::main
  10: std::rt::lang_start::{{closure}}
  11: std::panicking::try::do_call
             at src/libstd/rt.rs:49
             at src/libstd/panicking.rs:293
  12: __rust_maybe_catch_panic
             at src/libpanic_unwind/lib.rs:87
  13: std::rt::lang_start_internal
             at src/libstd/panicking.rs:272
             at src/libstd/panic.rs:388
             at src/libstd/rt.rs:48
  14: main
  15: __libc_start_main
  16: _start

```

**Comment by komaeda** [comment]
Can you post the output of `ls -a`?

**Comment by vn-ki** [comment]
```
$ ls -a                                                               
.git  .gitignore  .travis.yml  Cargo.lock  Cargo.toml  default_out.txt  exercises  info.toml  install.sh  LICENSE  README.md  src  target  tests
```

**Comment by komaeda** [comment]
@vn-ki Are you still experiencing this issue?

**Comment by justinbeal** [comment]
I'm also getting this on manjaro trying to edit with VSCode. I'm pasting what I think are the appropriate details here. 

`$ rustlings --version
rustlings 1.4.1
`

`$ RUST_BACKTRACE=1 rustlings watch
⠁ Compiling exercises/variables/variables1.rs...
thread 'main' panicked at 'Failed to run 'compile' command.: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/libcore/result.rs:999:5
stack backtrace:
   0: backtrace::backtrace::libunwind::trace
             at /cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.29/src/backtrace/libunwind.rs:88
   1: backtrace::backtrace::trace_unsynchronized
             at /cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.29/src/backtrace/mod.rs:66
   2: std::sys_common::backtrace::_print
             at src/libstd/sys_common/backtrace.rs:47
   3: std::sys_common::backtrace::print
             at src/libstd/sys_common/backtrace.rs:36
   4: std::panicking::default_hook::{{closure}}
             at src/libstd/panicking.rs:200
   5: std::panicking::default_hook
             at src/libstd/panicking.rs:214
   6: std::panicking::rust_panic_with_hook
             at src/libstd/panicking.rs:477
   7: std::panicking::continue_panic_fmt
             at src/libstd/panicking.rs:384
   8: rust_begin_unwind
             at src/libstd/panicking.rs:311
   9: core::panicking::panic_fmt
             at src/libcore/panicking.rs:85
  10: core::result::unwrap_failed
  11: rustlings::exercise::Exercise::compile
  12: rustlings::verify::compile_only
  13: rustlings::main
  14: std::rt::lang_start::{{closure}}
  15: std::rt::lang_start_internal::{{closure}}
             at src/libstd/rt.rs:49
  16: std::panicking::try::do_call
             at src/libstd/panicking.rs:296
  17: __rust_maybe_catch_panic
             at src/libpanic_unwind/lib.rs:82
  18: std::panicking::try
             at src/libstd/panicking.rs:275
  19: std::panic::catch_unwind
             at src/libstd/panic.rs:394
  20: std::rt::lang_start_internal
             at src/libstd/rt.rs:48
  21: main
  22: __libc_start_main
  23: _start
`

`$ ls -a
.  ..  Cargo.lock  Cargo.toml  CHANGELOG.md  .clog.toml  CONTRIBUTING.md  default_out.txt  exercises  .git  .gitignore  info.toml  install.sh  LICENSE  README.md  rustlings  src  target  tests  .travis.yml`

**Comment by fmoko** [comment]
Hopefully fixed by #233
